### PR TITLE
NEWS: Updated NEWS for 1.15.0-rc3.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Updated NEWS file filling 1.15.0-rc3 sections.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -11,6 +11,15 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc2 (August 9, 2023)
```
Comment (shamisp) [inline NEWS:14]: rc3 ?

**Review by shamisp — CHANGES_REQUESTED** [review]

File: NEWS [hunk 2]
```diff
@@ -11,6 +11,15 @@
...
+### Bugfixes:
+#### UCP
+* Fixed endpoint reconfiguration issues because of assymetrical selection
+#### UCM
+* Fixed conditional jump patching
+#### Tests
+* Multiple fixes for Azure CI pipeline
```
Comment (brminich) [inline NEWS:21]: maybe no need to mention as this is just internal infra

Comment (rakhmets) [reply NEWS]: removed

**Review by brminich — DISMISSED** [review]

File: NEWS [hunk 3]
```diff
@@ -11,6 +11,13 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc3 (August 9, 2023)
+### Bugfixes:
+#### UCP
+* Fixed endpoint reconfiguration issues because of assymetrical selection
+#### UCM
+* Fixed conditional jump patching
```
Comment (yosefe) [inline NEWS:19]:
```
#### UCT
 * Check dmabuf kernel support in ROCm memory domain
#### Tools
 * Fixed memory access flags in perftest 
```

File: NEWS [hunk 4]
```diff
@@ -11,6 +11,13 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc3 (August 9, 2023)
```
Comment (yosefe) [inline NEWS:14]: can set date to August 8

File: NEWS (hunk 3 above)
Comment (rakhmets) [reply NEWS:21]: Added

File: NEWS (hunk 4 above)
Comment (rakhmets) [reply NEWS]: updated

**Review by yosefe — APPROVED** [review]

**Comment by shamisp** [comment]
Looks good

**Review by shamisp — APPROVED** [review]

**Pull request merged**